### ISSUE TITLE
Make it easy for a person to contact 311 from their app

### ISSUE THREAD

**Issue by hackajesse** [sfbrigade/adopt-a-drain#41]
for drains that they can't clear or that they see need clearing or an already flooded area.

**Comment by jeanwalshie** [comment]
:thumbsup:

**Comment by jeanwalshie** [comment]
This is another idea I'd love to move forward. It would be good for a user to be able to report a problem with their drain right from the app. Currently there are links to 311 in the purple "learn what it means to adopt" box -- but what if there were a button in the pop up window for an adopted drain that said, "Report a Problem" or something? Maybe the link could go directly here: http://crmproxy.sfgov.org/Ef3/General.jsp?form=SSP_Street_Sidewalk_Cleaning&page=SSP_Page_EmailEnter

**Comment by jszwedko** [comment]
It looks like 311 exposes an API that we could possibly use to post directly from the app: http://wiki.open311.org/San_Francisco/#SF_Open_311_API

**Comment by hackajesse** [comment]
Riverside has connected their 311 app I believe.

Feeling like we should user test a bit before building more stuff...

**Comment by jeanwalshie** [comment]
@hackajesse yes, i agree. how do we make that happen? i've been testing like mad, but i don't really count since i'm too close to it.
